src/model/Description.ts

```diff
@@ -0,0 +1,25 @@
+export class DescriptionParameter {
+  readonly name: string
+  readonly description: string
+
+  constructor(name: string, description: string) {
+    this.name = name
+    this.description = description
+  }
+}
+
+export class DescriptionEntry {
+  readonly patternId: string
+  readonly title: string
+  readonly description: string
+  readonly timeToFix: number
+  readonly parameters?: DescriptionParameter[]
```

Comment:
not optional

Reply:
Same story as above, facility for parsing.